src/util/HttpServer/HttpUtils.h

```diff
@@ -71,6 +73,21 @@ static auto createOkResponse(std::string text, const HttpRequest auto& request,
                                       request, mimeType);
 }
 
+/// Create a HttpResponse from a string with status 200 OK. Otherwise behaves
+/// the same as createHttpResponseFromString.
+static auto createOkResponse(
+    ad_utility::stream_generator::stream_generator&& generator,
+    const HttpRequest auto& request, MediaType mimeType) {
+  http::response<ad_utility::httpUtils::httpStreams::streamable_body> response{
+      http::status::ok, request.version()};
+  response.set(http::field::content_type, toString(mimeType));
+  response.keep_alive(request.keep_alive());
+  response.body() = std::move(generator);
+  // Set Content-Length and Transfer-Encoding.
+  response.prepare_payload();
+  return response;
+}
```

Comment:
For my understanding (and maybe comment here):
How does beast determine, that the transfer-encoding is chunked?
Does it call `get` once, and determines if the bool is true, and then it knows ` at least two blocks-> transfered and no content length` ? Maybe state this here, it is important, since we don't have that much expertise yet with `beast`. (except maybe you).